File: homeworkAssignments/armstrongNumbers/armStrong.js
const armstrong = (num) => {
    let total = 0
    let str = num.toString()
    if (str.length < 2) {
        return `${num} is an armstrong number`
    }

    for (let number in str) {
        total += str[number] ** str.length
    }
    if (total === num) {
        return `${num} is an armstrong number`
    } else { return `${num} is not an armstrong number`}
}

console.log(armstrong(371))

File: homeworkAssignments/homeworkAssignments/armstrongNumbers/arm_strong.py
import math

def armstrong(stop):
    armstrong_list = []
    
    for num in range(stop):
        total = 0
        arr = str(num)
        # Verify number is an armstrong number
        for number in arr:
            total += math.pow(int(number),len(arr))
        if total == num:
            armstrong_list.append(num)
    return armstrong_list

print(armstrong(999))
